README.md

```diff
@@ -32,13 +32,16 @@ Q2: What is the answer?
 Hello!  In the rush that is the start of the term, I really haven't had a chance to chat with many of you.  So would you do me the favor of helping me get to know a bit about you by answering the following?  Please avoid one word answers... I want to learn about you!
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
+* _Mark_: I have always been interested in programming since my first experiences in middle school on TRS-80 machines. My senior year in high school I took a programming class, which was a brand new subject at the time. I wrote a quadratic formula program in Microsoft Extended Color BASIC on my TRS-80 Color Computer for extra credit, and the teacher accused me of plagiarizing. Not that I needed the extra credit, but I did have to defend myself on that.
 
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
+* _Mark_ : I'm not that big into travel. I like the feeling of home. As a college student I did live in France for six months - actually turned 21 in France. I would very much like to visit Montreal (again), Belgium, and Brazil.
```

Comment:
Where in France did you live? Please add the answer to your interview and update your pull request.